BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -209,6 +213,80 @@ private void ConvertFuncStatic(Entity funcStatic)
 			funcStatic.ClassName = "func_brush";
 		}
 
+		private void ConvertFuncPlat(Entity entity)
+		{
+			var moveDistance = 0f;
+			var brushThickness = GetBrushThickness(entity);
+
+			if (float.TryParse(entity["height"], out var height))
+				moveDistance = height + brushThickness;
+			else if (float.TryParse(entity["lip"], out var lip))
+				moveDistance = -(lip - 2 - (brushThickness * 2));
+
+			if (string.IsNullOrEmpty(entity.Name))
+			{
+				entity.Name = $"plat{platIndex}";
+				CreatePlatTrigger(entity);
```

Comment:
This trigger won't get created if the `func_plat` has an existing name. Is this intended?

Reply:
I was going off the q3 source code, but I might be misinterpreting it.

``` 
	if ( !ent->targetname ) {
		SpawnPlatTrigger(ent);
```

Looks like the trigger is only made when the entity doesnt have a targetname.